test/unit/smart_answer_flows/calculate_statutory_sick_pay_flow_test.rb

```diff
@@ -46,58 +46,67 @@ class CalculateStatutorySickPayFlowTest < ActiveSupport::TestCase
 
```

Comment:
Would you be able to update the commit note to explain why you've used a real instance instead of a stub?